packages/audit/scripts/kpi-report.py

```
@@ -0,0 +1,157 @@
+    qg = scorecard["quality_gates"]
+    lines.append("## Quality Gates")
+    lines.append(f"- Total: {qg['total']}")
+    lines.append(f"- Passed: {qg['passed']}")
+
+    return "\n".join(lines) + "\n"
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser(description="KPI scorecard for ai-orchestra audit logs")
+    parser.add_argument("--days", type=int, default=None, help="直近 N 日間で集計")
+    parser.add_argument("--output", help="出力ファイル（省略時は stdout）")
```

Ruff (0.15.9)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)